src/main/java/seedu/address/ui/InternshipCard.java

```diff
@@ -48,9 +50,21 @@ public InternshipCard(Internship internship, int displayedIndex) {
         role.setText(internship.getInternshipRole().roleName);
         status.getChildren().add(new Label(internship.getInternshipStatus().toString()));
         if (internship.getContactPersonId() == null) {
-            contactPersonId.setText("No contact person.");
+            contactPerson.setText("No contact person.");
```

Comment:
This could have just been NO_CONTACT_PERSON

Reply:
changed